src/lib.rs

```diff
@@ -259,15 +259,34 @@ fn serve_request_stream<Ctx: Clone>(
 ) -> impl Stream<Item = Result<Message, warp::Error>> {
     let resp_stream = srv
         .serve_ws(ctx, payload, service_id)
-        .map(move |payload_result| match payload_result {
-            Ok(payload) => Outgoing::Next {
-                request_id: req_id,
-                payload,
-            },
-            Err(kind) => Outgoing::Error {
-                request_id: req_id,
-                kind,
-            },
+        .take_until_condition(|resp| future::ready(resp.is_err()))
+        .ready_chunks(128)
```

Comment:
make config?

Reply:
I wound not bother. Keep it as is for now, and then make it configurable once we got a proper mechanism in place. The mechanism needs to take into consideration not just count but also size. E.g. if you have 128 100kb events, you probably just want to send them one by one.